superset-embedded-sdk/src/index.ts

```diff
@@ -99,6 +102,9 @@ export async function embedDashboard({
     return new Promise(resolve => {
       const iframe = document.createElement('iframe');
       const dashboardConfig = dashboardUiConfig ? `?uiConfig=${calculateConfig()}` : ""
+      const queryParams = queryParameters ?
+        (dashboardConfig === "" ? "?" : "&") + new URLSearchParams(queryParameters).toString() :
+        ""
```

Comment:
nit: this feels slightly hacky. Could we perhaps create a new object `allQueryParams` like this:

```javascript
const allQueryParams = {...queryParameters}
if (dashboardUiConfig) {
    allQueryParams['uiConfig'] = calculateConfig();
}
```
and then just do 

```javascript
iframe.src = `${supersetDomain}/embedded/${id}{new URLSearchParams(allQueryParams).toString()}`;
```

(note: I didn't test this, so consider this javascript-y pseudocode)